CHANGELOG.md

```diff
@@ -102,6 +102,17 @@ If you're importing JavaScript modules individually, you should check any refere
 
 This change was introduced in [pull request #1836: Rename exported JavaScript modules to include component name](https://github.com/alphagov/govuk-frontend/issues/1836)].
 
+#### Update the HTML for hints
+
+In v3.8.0, we updated the HTML for hints to use a <div> rather than a <span> element, to allow block elements to be used.
+We have now removed the styles which made the hint <span> display as block.
+
+Update your hint HTML to use a <div> rather than a <span>, for example: `<div id="input-hint" class="govuk-hint"></div>`, to ensure your hints are styled correctly.
```

Comment:
```suggestion
To make sure your hints are styled correctly, update your hint HTML to use a `<div>` rather than a `<span>`. For example: `<div id="input-hint" class="govuk-hint"></div>`.
```